pyveg/paper/paper.md

```diff
@@ -129,6 +135,12 @@ Early warning signals are also computed using the `ewstools` package [@ewstools]
 including Lag-1 autocorrelation and standard deviation moving window plots.
 A sensitivity and significance analysis is also performed in order to determine
 whether any trends (quantified by Kendall tau values) are statistically significant.
+The vegetation return rate is calculated by fitting the crystall ball function
```

Comment:
I would change "fitting the crystall ball function" to "fitting a crystall ball function" and I would add a reference to it given that is not a known term, maybe the wikipedia should be enough:
“Crystal Ball Function”, https://en.wikipedia.org/wiki/Crystal_Ball_function